app/models/cart.rb

```diff
@@ -16,5 +16,6 @@ def count(data)
 
   def remove_item(item_id)
     @data[item_id] -= 1
+    @data.delete_if { |item, quantity| quantity == 0 }
```

Comment:
Unused block argument - `item`. If it's necessary, use `_` or `_item` as an argument name to indicate that it won't be used.